draft-terbu-sd-jwt-vc.md

```diff
@@ -157,6 +157,39 @@ VC-SD-JWT with the following components:
 1. An SD-JWT to protect the integrity of the claims, to enable selective disclosure and to ensure authorship of the VC-SD-JWT. The SD-JWT header parameters and the payload is further defined by this specification.
 1. The full set of SD-JWT Disclosures that contain the claims in plain text.
 
+# Flow Diagram
+~~~ ascii-art
+           +------------+
+           |            |
+           |   Issuer   |
+           |            |
+           +------------+
+                 |
+             Issues SD-JWT
+     and Issuer-Issued Disclosures
+                 |
+                 v
+           +------------+                                 +------------+ 
+           |            |                                 |   Status   |
+           |   Holder   |-------- optionally ------------>|  Provider  |
+           |            |       retrieves status          |            |
+           +------------+             VC                  +------------+ 
+                 |
```

Comment:
We did not introduce the status provider yet. If we want to have it here (can make sense), let's introduce it somewhere first.

Reply:
Added it to the terminology section.